src/openms/include/OpenMS/ANALYSIS/SVM/SimpleSVM.h

```diff
@@ -45,10 +45,10 @@
 namespace OpenMS
 {
   /**
-     @brief Simple interface to support vector machines for classification (via LIBSVM).
+     @brief Simple interface to support vector machines for classification and regression (via LIBSVM).
 
-     This class supports (multi-class) classification with a linear or RBF kernel.
-     It uses cross-validation to optimize the SVM parameters @e C and (RBF kernel only) @e gamma.
+     This class supports (multi-class) classification with a linear or RBF kernel and regression with an epsilon-SVR.
```

Comment:
The two kernels are applicable to SVR as well, right?
If yes, change to: "... classification and regression (epsilon-SVR) with a linear or RBF kernel"